docs/planning-experiments/index.md

```diff
@@ -0,0 +1,6 @@
+# Planning Experiments
+
+- [Minimum Detectable Effects](./minimum_detectable_effects)
+- [Entry Points](./entry_points)
```

Comment:
This is out of order with the pages in the nav bar, feels a bit jarring

Reply:
Fixed!